### DISCUSSION TITLE
Track User Traffic From Email Campaigns

### DISCUSSION THREAD

**Discussion by dstearle** [umami-software/umami#976] [Q&A]
Maybe this is far more simple than I have imagined (or more complicated) but is there a way to track users coming to our site via email campaigns? For example, a user receives an email to visit our site and clicks it, that shows up somewhere under "referrers" on the dashboard. I am reading on google analytics and I guess this would be done with a "trackable url". Is there currently any feature on Umami that does this or a way to achieve a similar result with what is available?

**Comment by mikecao** [comment]
You would need to add query parameters to your url, like https://en.wikipedia.org/wiki/UTM_parameters.

**Reply by dstearle** [reply]
Thanks that helped a lot! From what I see is that in order to view the amount of hits from something like "https://test.com/?utm_source=email" I would have to go to "pages" for whatever website I have on Umami, then view under the "Raw" setting directly above the list.
